app/models/manageiq/providers/ansible_playbook_workflow.rb

```diff
@@ -17,9 +18,9 @@ def pre_playbook
   end
 
   def run_playbook
-    credentials, env_vars, extra_vars, hosts, playbook_path = options.values_at(:credentials, :env_vars, :extra_vars, :hosts, :playbook_path)
+    credentials, env_vars, extra_vars, hosts, playbook_path, verbosity = options.values_at(:credentials, :env_vars, :extra_vars, :hosts, :playbook_path, :verbosity)
 
-    response = Ansible::Runner.run_async(env_vars, extra_vars, playbook_path, :hosts => hosts, :credentials => credentials)
+    response = Ansible::Runner.run_async(env_vars, extra_vars, playbook_path, :hosts => hosts, :credentials => credentials, :verbosity => verbosity)
```

Comment:
You might be able to get slightly tighter code here with something like

```ruby
env_vars, extra_vars, playbook_path = options.values_at(:env_vars, :extra_vars, :playbook_path)
kw_args = options.slice(:credentials, :hosts, :verbosity)

response = Ansible::Runner.run_async(env_vars, extra_vars, playbook_path, kw_args)
```

Reply:
That's coming in my next patch :laughing: https://github.com/carbonin/manageiq/compare/add_more_options_to_ansible_stuff...carbonin:respect_escalate_privilege_setting?expand=1#diff-b975cd0ffa42d0fa086105cec05d6caeR23